pynestml/codegeneration/nest_codegenerator.py

```diff
@@ -850,8 +853,9 @@ def log_set_source_position(node):
 
         return inline_expressions
 
-    def replace_inline_expressions_through_defining_expressions(self, definitions, inline_expressions):
-        # type: (list(ASTOdeEquation), list(ASTInlineExpression)) -> list(ASTInlineExpression)
+    def replace_inline_expressions_through_defining_expressions(self,
+                                                                definitions: Sequence[ASTOdeEquation],
+                                                                inline_expressions: Sequence[ASTInlineExpression]) -> Sequence[ASTOdeEquation]:
```

Comment:
Should the return type here be `Sequence[ASTInlineExpression]`?

Reply:
The returned object is the same as the first function parameter, so is correct. However, I updated the outdated docstring.